Jenkinsfile

```diff
@@ -17,6 +17,9 @@ def pipeline = new org.js.AppPipeline(steps: this,
     copyStaticToBranch: true,
     copyToBranches: ['fleek-pre', 'fleek'],
     copyFile: 'env.json',
+    ipfsHashNotification: true,
+    fleekDefaultSiteName: 'long-firefly-8047',
```

Comment:
Need to replace it with polkaswap2.io

Reply:
Request allows only fleek's default names, we can't use custom name here